src/app/containers/Config/Config.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { TabBar, Tab } from 'rmwc/Tabs'
+import '@material/tabs/dist/mdc.tabs.min.css'
+
+import CustomNetworkList from '../../components/CustomNetworkList'
+import AddCustomNetwork from '../../components/AddCustomNetwork'
+
+import style from './Config.css'
+
+class Config extends Component {
+  state = {
+    activeTabIndex: 0,
+  }
+
+  render() {
+    const { activeTabIndex } = this.state
+    const { addCustomNetwork, deleteCustomNetwork, setNetwork, networks, selectedNetworkId } = this.props
+
+    let tabContent = null
+    if (activeTabIndex === 1) {
+      tabContent = <AddCustomNetwork addCustomNetwork={ addCustomNetwork } />
+    } else {
+      tabContent = <CustomNetworkList
+        deleteCustomNetwork={ deleteCustomNetwork }
+        setNetwork={ setNetwork }
+        selectedNetworkId={ selectedNetworkId }
+        networks={ networks }
+      />
+    }
+
+    return (
+      <div>
+        <TabBar
+          className={ style.tabBar }
+          activeTabIndex={ this.state.activeTabIndex || 0 }
+          onChange={ evt => this.setState({ 'activeTabIndex': evt.target.value }) }
+        >
+          <Tab style={ { display: 'table-cell' } }>Networks</Tab>
```

Comment:
Lets use css classes for any styling we need to do. In addition table-cell isn't the best for layout, have you considered a flexbox alternative?